tzlocal/unix.py

```diff
@@ -35,12 +45,9 @@ def _get_localzone(_root='/'):
     beneath the _root directory. This is primarily used by the tests.
     In normal usage you call the function without parameters."""
 
-    tzenv = os.environ.get('TZ')
+    tzenv = pytz.unix._try_tz_from_venv()
```

Comment:
This is wrong, which is why the tests fail. It should be just `_try_tz_from_venv()`